sdk/python/using-mlflow/model-management/model_management.ipynb

```diff
@@ -280,10 +301,13 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can query all the registered models in the registry using the MLflow client with the method `list_registered_models`."
+    "You can query all the registered models in the registry using the MLflow client with the method `search_registered_models`.\n",
+    "\n",
+    "> __MLflow 2.0 advisory__: In older versions of Mlflow (<2.0), please use `list_registered_models` instead."
```

Comment:
re-word to "search_registered_models is recommended instead of list_registered_models" ?

https://pyup.io/packages/pypi/mlflow/changelog
MLflow 1.10.0 - Added paginated `MlflowClient.search_registered_models` API (2939, 3023, 3027 ankitmathur-db; 2966, mparkhe)